### ISSUE TITLE
ng-style with background image

### ISSUE THREAD

**Issue by Valonix** [revolunet/angular-carousel#418]
Hi there.
Can i use something like this?

`
<ul class="slider__list" rn-carousel
                    rn-carousel-index="$ctrl.carouselIndex"
                    rn-carousel-controls
                    rn-carousel-controls-allow-loop
                >
                    <li ng-repeat="slide in $ctrl.slides track by slide.id"
                        class="slide"
                        ng-style="{'background-image': 'url(/images/slides/' + slide.image + ')'}">
                        <div class="slide-wrap">
                            <img class="img-responsive" ng-src="/images/logos/{{$ctrl.logo}}.svg" alt="">
                            <p>{{slide.body}}</p>
                        </div>
                    </li>
                </ul>
`

Because right now i didn`t see backgound image..

**Comment by Valonix** [comment]
so? any ideas?

**Comment by auxcoder** [comment]
Yes you can but `ng-style` accept an expression, should be like this `ng-style="{'background-image': 'url(/images/slides/{{ slide.image}})'}"`.
